h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -211,6 +212,8 @@ public GLMModel clone2(){
     public boolean _stdOverride; // standardization override by beta constraints
 
     public void validate(GLM glm) {
+      if (_solver.equals(Solver.COORDINATE_DESCENT_NAIVE))
+        throw H2O.unimpl("Naive coordinate descent is not supported.");
```

Comment:
Is COORDINATE_DESCENT_NAIVE not supported for multinomial?  Or for families as well?

Reply:
This has been removed in the next commit.